lang/lib/FontGen.py

```diff
@@ -34,45 +36,97 @@ def __init__(self, utf, charListIndex, alternate = None):
     CustomCharacter('ă', 23, 'a'),
     CustomCharacter('Ă', 31, 'A'),
     CustomCharacter('ć', 32, 'c'),
-    CustomCharacter('ć', 32, 'c'),
+    # CustomCharacter('Ć', 36, 'C'),
+    # CustomCharacter('ç', 33, 'c'),
+    # CustomCharacter('Ç', 37, 'C'),
     CustomCharacter('č', 34, 'c'),
+    CustomCharacter('Č', 38, 'C'),
     CustomCharacter('ď', 40, 'd'),
+    # CustomCharacter('Ď', 42, 'D'),
+    # CustomCharacter('đ', 41, 'd'),
+    # CustomCharacter('Đ', 43, 'D'),
     CustomCharacter('é', 44, 'e'),
+    CustomCharacter('É', 52, 'E'),
     CustomCharacter('è', 45, 'e'),
+    # CustomCharacter('È', 53, 'E'),
+    CustomCharacter('ê', 46, 'e'),
+    # CustomCharacter('Ê', 54, 'E'),
     CustomCharacter('ě', 47, 'e'),
+    # CustomCharacter('Ě', 55, 'E'),
+    CustomCharacter('ë', 48, 'e'),
+    # CustomCharacter('Ë', 56, 'E'),
     CustomCharacter('ę', 49, 'e'),
-    CustomCharacter('É', 52, 'E'),
+    CustomCharacter('Ę', 57, 'E'),
     CustomCharacter('í', 60, 'i'),
+    CustomCharacter('Í', 64, 'I'),
+    # CustomCharacter('ì', 61, 'i'),
+    # CustomCharacter('Ì', 65, 'I'),
     CustomCharacter('î', 62, 'i'),
     CustomCharacter('Î', 66, 'I'),
     CustomCharacter('ĺ', 68, 'l'),
+    # CustomCharacter('Ĺ', 72, 'L'),
     CustomCharacter('ł', 69, 'l'),
-    CustomCharacter('ľ', 70, 'l'),
     CustomCharacter('Ł', 73, 'L'),
+    CustomCharacter('ľ', 70, 'l'),
     CustomCharacter('Ľ', 74, 'L'),
     CustomCharacter('ń', 76, 'n'),
+    # CustomCharacter('Ń', 80, 'N'),
     CustomCharacter('ň', 77, 'n'),
-    CustomCharacter('ñ', 78, 'n'),
+    # CustomCharacter('Ň', 81, 'N'),
+    # CustomCharacter('ñ', 78, 'n'),
+    # CustomCharacter('Ñ', 82, 'N'),
     CustomCharacter('ó', 84, 'o'),
+    CustomCharacter('Ó', 92, 'O'),
     CustomCharacter('ò', 85, 'o'),
+    # CustomCharacter('Ò', 93, 'O'),
     CustomCharacter('ô', 86, 'o'),
+    # CustomCharacter('Ô', 94, 'O'),
+    # CustomCharacter('œ', 87, 'o'),
+    # CustomCharacter('Œ', 95, 'O'),
     CustomCharacter('ø', 88, 'o'),
+    CustomCharacter('Ø', 96, 'o'),
+    CustomCharacter('Ö', 97, '\\xef'),
     CustomCharacter('ő', 90, 'o'),
+    # CustomCharacter('Ő', 98, 'O'),
+    # CustomCharacter('ŕ', 100, 'r'),
+    # CustomCharacter('Ŕ', 104, 'R'),
     CustomCharacter('ř', 101, 'r'),
+    CustomCharacter('Ř', 105, 'R'),
     CustomCharacter('ś', 108, 's'),
+    CustomCharacter('Ś', 112, 'S'),
     CustomCharacter('š', 109, 's'),
+    CustomCharacter('Š', 113, 'S'),
     CustomCharacter('ș', 110, 's'),
+    CustomCharacter('Ș', 114, 'S'),
     CustomCharacter('ß', 111, '\\xe2'),
+    # CustomCharacter('ẞ', 115, '\\xe2'),
```

Comment:
Why is `ß` 111 and 115 and one active and the other inactive?

Reply:
uppercase version of ß. Should only ever be used in all-caps words such as MMU errors, but it isn't used anywhere. But since I have it in the font file, I added it to FontGen.py for future reference.

Reply:
yes

Reply:
https://en.wikipedia.org/wiki/%C3%9F#:~:text=A%20second%20proposal,orthography%20in%202017.